spec/graphql/mutations/stories/delete_story_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+module Mutations
+  module Stories
+    RSpec.describe DeleteStory, type: :request do
+      def query
+        <<~GQL
+          mutation {
+          deleteStory(
+            input: {
+              id: "#{@story.id}",
+            }
+          ) { story {
+              id
+              title
+            }
+          }
+        }
+        GQL
+      end
+
+      def bad_query
+        <<~GQL
+          mutation {
+          deleteStory(
+            input: {
+              id: "10900",
+            }
+          ) { story {
+              id
+              title
+            }
+          }
+        }
+        GQL
+      end
+
+      it 'deletes a story' do
+        stories = create_list(:story, 5)
+        @story = create(:story, title: 'Deleted')
+
+        post '/graphql', params: { query: query }
+
+        response = parse_json[:data]
+        expect(response[:deleteStory][:story][:title]).to eq('Deleted')
+        expect(Story.last.title).to_not eq('Deleted')
+      end
+
+      it 'cant find story to delete' do
+        stories = create_list(:story, 5)
```

Comment:
Curious if this needs to be set to a variable since we don't call it later. Could we not just create stories here witout setting it to a variable?

Reply:
It doesn't need to be a variable, just added extra stories.